osquery/tables/system/linux/system_info.cpp

```diff
@@ -57,9 +59,17 @@ QueryData genSystemInfo(QueryContext& context) {
     r["physical_memory"] = "-1";
   }
 
-  r["cpu_type"] = "0";
   r["cpu_subtype"] = "0";
 
+  // https://github.com/karelzak/util-linux/blob/7085f1e49b0a2a670ba068705da6084d19a41a5a/sys-utils/lscpu.c#L394-L400
+  struct utsname utsbuf;
+  if (uname(&utsbuf) == -1) {
+    VLOG(1) << "Error: uname failed";
```

Comment:
Be more descriptive, like, "Failed to get architecture, uname failed with error code: " << std::to_string(errno);